### ISSUE TITLE
Vis Resonator Crafting

### ISSUE THREAD

**Issue by AlvArt369** [Azanor/thaumcraft-suggestions#451]
There's no way to get to craft the Vis Resonator :'I 

![2018-10-16_13 23 00](https://user-images.githubusercontent.com/44209790/47038297-b9356400-d146-11e8-88aa-cd2b0c7aa80e.png)

**Comment by dmarcuse** [comment]
You need to craft it in an arcane workbench, it takes vis if I remember correctly.

**Comment by AlvArt369** [comment]
Yeah, I tried that, it seems to use aer and aqua vis but it's not working either. At first I thought the item just didn't show, but as far as I know you just can't craft it even though it has a recipe.

![2018-10-16_16 19 36](https://user-images.githubusercontent.com/44209790/47048218-58665580-d15f-11e8-98c6-1da7867b834c.png)
![2018-10-16_16 19 48](https://user-images.githubusercontent.com/44209790/47048226-5dc3a000-d15f-11e8-89cb-e9173eecbbff.png)

**Comment by dmarcuse** [comment]
Check the recipe in the thaumonomicon, it might take more than one of a type of crystal
